tests/unit/onyxTest.js

```diff
@@ -358,6 +358,89 @@ describe('Onyx', () => {
             });
     });
 
+    it('should use update data object to merge a collection of keys', () => {
+        const valuesReceived = {};
+        const mockCallback = jest.fn(data => valuesReceived[data.ID] = data.value);
+        connectionID = Onyx.connect({
+            key: ONYX_KEYS.COLLECTION.TEST_KEY,
+            initWithStoredValues: false,
+            callback: mockCallback,
+        });
+
+        return waitForPromisesToResolve()
+            .then(() => {
+                // GIVEN the initial Onyx state: {test_1: {existingData: 'test',},test_2: {existingData: 'test',}}
```

Comment:
NAB: OCD
```suggestion
                // GIVEN the initial Onyx state: {test_1: {existingData: 'test',}, test_2: {existingData: 'test',}}
```

Reply:
Fixed. Now you can sleep happily tonight :D